pkgs/os-specific/linux/kernel/bcachefs.json

```
@@ -0,0 +1,5 @@
+{
+  "diffHash": "sha256-BmXd/5Hn/xr7gNFp6BsBMG2XeKFlPGxv66IQ8DEwh5k=",
```

I got something else here:

```suggestion
  "diffHash": "sha256-DtMc8P4lTRzvS6PVvD7WtWEPsfnxIXSpqMsKKWs+edI=",
```